toolchain/lc-compile/src/set.c

```diff
@@ -22,7 +22,7 @@ typedef unsigned int SetRef;
 
 #define MAX_SET_DEPTH 256
 SetRef s_sets[MAX_SET_DEPTH];
-unsigned int s_set_count = 0;
```

Comment:
Thanks @peter-b. I guess I was confused as to why the APIs used signed parameters as it wouldn't appear to be valid to use them with negative parameters.